zookeeper-server/src/main/java/org/apache/zookeeper/ZKUtil.java

```diff
@@ -290,4 +291,4 @@ public static String aclToString(List<ACL> acls) {
         }
         return sb.toString();
     }
-}
\ No newline at end of file
+}
```

Comment:
Here is a useless change?

Reply:
Seems it's my IDE's auto-format. I'will revert it.

Reply:
Can't figure it out, I've compare this line and it's upper with master, no difference found.